packages/opentelemetry-exporter-collector-grpc/README.md

```diff
@@ -0,0 +1,141 @@
+# OpenTelemetry Collector Exporter for node with grpc
+
+[![Gitter chat][gitter-image]][gitter-url]
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+This module provides exporter for web and node to be used with [opentelemetry-collector][opentelemetry-collector-url] - last tested with version **0.6.0**.
```

Comment:
I tested with the collector v0.8.0 for both metrics and traces and the exporter works, if you want to update this

Reply:
Update: there are some errors with the metrics exporter and collector v0.8.0, i'm trying to pinpoint them

Reply:
It might be as collector is changing all the time as well as the proto. I would not change it yet this information, but if there are anything we need to fix I would make a new PR after this will be merged, otherwise it will be difficult to see what are the changes / fixes.